Domains/4-Application/DrawingProduction/SheetNumberingViews.ecschema.xml

```diff
@@ -0,0 +1,189 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- ==================================================================================
+|  * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+|  * See LICENSE.md in the project root for license terms and full copyright notice.
+======================================================================================= -->
+<ECSchema schemaName="SheetNumberingViews" alias="snumViews" version="01.00.00" xmlns="http://www.bentley.com/schemas/Bentley.ECXML.3.2" description="Contains classes and relationships for sheet numbering.">
+    <ECSchemaReference name="CoreCustomAttributes" version="01.00.03" alias="CoreCA" />
+    <ECSchemaReference name="BisCustomAttributes" version="01.00.00" alias="bisCA"/>
+    <ECSchemaReference name="BisCore" version="01.00.16" alias="bis"/>   
+    <ECSchemaReference name="ECDbMap" version="02.00.04" alias="ecdbmap"/>
+
+    <ECCustomAttributes>
+        <ProductionStatus xmlns="CoreCustomAttributes.01.00.03">
+            <SupportedUse>NotForProduction</SupportedUse>
+        </ProductionStatus>
+        <SchemaLayerInfo xmlns="BisCustomAttributes.01.00.00">
+            <Value>Application</Value>
+        </SchemaLayerInfo>
+    </ECCustomAttributes>
+
+    <ECEntityClass typeName="SheetIndexContainerView" modifier="Abstract" displayLabel="SheetIndex Container View" description="View that returns the number of children under a parent. This can be either a SheetIndex or SheetIndexFolder.">
```

Comment:
Just wanted to double-check that you intentionally preferred the name `SheetIndexContainer`. It sounds to me like a "Container for Sheet-Indexes", which would be semantically incorrect - but if you applied a different rationale, that's fine.

Otherwise, `SheetReferenceContainer` seems more semantically correct (i.e. Container for SheetReferences).

Reply:
I was following the pattern set by SheetIndexFolder.  While, in a vacuum, I agree with you, I prefer `SheetIndexContainer` given that context of `SheetIndexFolder`.